op-service/txmgr/txmgr.go

```diff
@@ -130,6 +130,10 @@ func (m *SimpleTxManager) BlockNumber(ctx context.Context) (uint64, error) {
 	return m.backend.BlockNumber(ctx)
 }
 
+func (m *SimpleTxManager) Sign(ctx context.Context, from common.Address, rawTx *types.DynamicFeeTx) (*types.Transaction, error) {
```

Comment:
I don't especially have a problem with this helper, but it's only invoked twice, and doesn't show any obvious benefit over the inline version.  Seems like it would be better to leave as is rather than diverge from upstream?

Reply:
Removed